src/prepack-standalone.js

```diff
@@ -30,17 +30,20 @@ import { Modules } from "./utils/modules.js";
 import { Logger } from "./utils/logger.js";
 import { Generator } from "./utils/generator.js";
 import { AbstractObjectValue, AbstractValue, ObjectValue } from "./values/index.js";
-import type { DebuggerConfigArguments } from "./types";
 
 export function prepackSources(
```

Comment:
This is a breaking change to our public package API. Rather put the new optional argument at the end, also making it optional.

Reply:
Sorry, I don't understand your suggestion:
> Rather put *the new optional argument* at the end

There is no new argument -- I'm removing `debuggerConfigArgs`. Are you saying to add a dummy optional argument to the end of the PrepackSources parameters?

Reply:
@NTillmann ^^